Arrangement_on_surface_2/include/CGAL/Arr_curve_data_traits_2.h

```diff
@@ -27,7 +27,6 @@
 #include <boost/utility/enable_if.hpp>
 #include <boost/mpl/has_xxx.hpp>
 
-#include <CGAL/Object.h>
 #include <CGAL/tags.h>
 #include <CGAL/Arr_tags.h>
 #include <CGAL/Arr_geometry_traits/Curve_data_aux.h>
```

Comment:
This header requires `<CGAL/assertions.h>`. See the Travis error:
https://travis-ci.com/github/CGAL/cgal/jobs/357577087#L7296

That header was probably pulled by `<CGAL/Object.h>`.